internal/kokoro/environment.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A suite of tests offered by https://github.com/googleapis/env-tests-logging
+# That allows deploying and testing features in live GCP services.
+# Currently only configured to test logging & error reporting
+
+set -eox pipefail
+
+# Test prechecks
+if [[ -z "${ENVIRONMENT:-}" ]]; then
+  echo "ENVIRONMENT not set. Exiting"
+  exit 1
+fi
+
+if [[ -z "${PROJECT_ROOT:-}"  ]]; then
+    PROJECT_ROOT="github/google-cloud-go"
+fi
+
+# Add the test module as a submodule to the repo.
+cd "${KOKORO_ARTIFACTS_DIR}/github/google-cloud-go/internal/"
+git submodule add https://github.com/googleapis/env-tests-logging
+cd "env-tests-logging/"
+
+# Disable buffering, so that the logs stream through.
+export PYTHONUNBUFFERED=1
+
+# Debug: show build environment
+env | grep KOKORO
+
+# Set up service account credentials
+export GOOGLE_APPLICATION_CREDENTIALS=$KOKORO_KEYSTORE_DIR/72523_go_integration_service_account
```

Comment:
nit: Nothing that get logged here is secret today, but I would still just just suggest moving this to the top before set -x is enabled in case something more sensitive shows up in the future.

Reply:
Sounds good. Tis moved. 

@codyoss Are you ok with my using the following?

1. I'm using go_integration_service_account from keystore. I may need to add IAM permissions to deploy CloudRun and CloudFunctions (required by the tests). Is that ok or should I create a new SA for this?

2. I'm putting this environment script under /kokoro directory. Let me know if you want me to nest it somewhere else!

Reply:
1. Yes!
2. Yes!

Reply:
Thanks for the PR!